pkg/apis/velero/v2alpha1/data_download_types.go

```diff
@@ -131,6 +131,7 @@ type DataDownloadStatus struct {
 // +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp",description="Time duration since this DataDownload was created"
 // +kubebuilder:printcolumn:name="Node",type="string",JSONPath=".status.node",description="Name of the node where the DataDownload is processed"
 
+// DataDownload acts as the protocol between data mover plugins and data mover controller for the datamover restore operation
```

Comment:
One thing I would add is that users should not create it, Velero creates it

Reply:
User actions aren't referenced in any of the description markers for the CRDs -- that's something that is probably better handled in the user documentation. Also, the description that was added implies that it's not user-managed in any case -- there are several velero CRDs that are like this, it's not unique to these two.